### ISSUE TITLE
Shield TV Crash Every Exit?

### ISSUE THREAD

**Issue by bdpna** [koying/SPMC#684]
Fresh install of 16.5.5 (did uninstall using Shield TV's trash can first).

Fresh install of 16.5.5 through the App Store still throwing the "SPMC has stopped" error dialog in the SATV interface every exit.

Anything I can provide log-wise etc?  Not sure if the logs will show anything or not since I get the "SPMC has stopped" message when I shutdown using the shutdown command, instead of the graceful exit back to the Shield interface.

I'm starting to wonder if this is normal behavior (seeing a stopped message) and if it is, if I can suppress it.  I suppose the other option is just always leave SPMC running and never shut it down.

**Comment by JFG90** [comment]
Others have reported this issue along with it crashing on start up along with random pausing, hopefully it's fixed on the next update

**Comment by MajorPump** [comment]
It's annoying but I do wonder why it's not been fixed yet as tons of people report it across various forums.  @Koying??

Sorry not having a dig, just wondering the root of the crash and what do you need from us to fix it? would normal logs work as it's after SPMC has closed

**Comment by JFG90** [comment]
I am now getting this every time i exit! So annoying :(

**Comment by JFG90** [comment]
I am now getting this every time i exit! So annoying :(

**Comment by Grindstolpe** [comment]
Also having the problem that SPMC 16.5-5 crashes on every exit, running on Shield ´17 with latest update.

It´s very annoying :(

**Comment by 99rook99** [comment]
Not annoying enough to provide a log, I guess.

**Comment by MajorPump** [comment]
So as I asked above how would a log issued during use help as the crash is after exit and if it is after exit how do we upload that?? please enlighten us, I am more than willing to supply any information that may help but as my question as to how to help was ignored there is nothing we can do to help. It seems the problem would rather be ignored for bigger and better things.

**Comment by 99rook99** [comment]
I  would say, if koying can't reproduce it a log would help. You can use the SPMC log uploader add on to upload the log. Just post the log id here so koying knows what one to look at.

**Comment by Grindstolpe** [comment]
Here´s a log id: pqbksvuma

But it doesn´t show any of my recent activities. Installed the latest test version and it also crashes, see the log.

Everytime I´ve watched a movie SPMC crashes, but my log-directory is completely empty even though I activated debugging.

**Comment by MajorPump** [comment]
Well I've been trying to upload a log all night and I just get "SPMC Log Uploader Error" and then refers me to view the log for the exact nature of the error so I'm getting no where.

**Comment by 99rook99** [comment]
@koying would have to help now,  since once a log fails I need help

**Comment by MajorPump** [comment]
OK, now have a log by doing it manually from System>Logging>Upload debug log...

http://paste.semperpax.com/pfu0dhz7f

I'm using Aeon Nox skin and only plugin installed is the official Plex one

I have now added what should be the above log but with the shutdown events. I pulled the log from the Shield with ADB and manually pasted it.

http://paste.semperpax.com/ppwdguez5
_

**Comment by MajorPump** [comment]
Right, I can now confirm on a factory reset Shield with a fresh install of SPMC 16.5.5 from the Play store that changing the skin appears to be the problem. All the while I use Confluence it exits fine. The moment I change to Aeon Nox (and also confirmed this with Amber) the error occurs, if I change back to Confluence the exit error goes away again.

:Edit****
@koying 
I can expand on the above now, more accurately it appears that using resources I have on the network and connecting via NFS is whats causing the crash on exit issue, if I swap to using the resources by SMB it goes away completely. With the logs above and this info I hope that helps.

Cheers for all the work you do SPMC, we couldn't do without it.
